frontend/README.md

```diff
@@ -22,3 +22,5 @@ npm run lint
 
 ### Customize configuration
 See [Configuration Reference](https://cli.vuejs.org/config/).
+
+dasdasd
```

Comment:
This seems like some garbage that wasn't meant to be committed, pls remove

Reply:
true, thanks for noticing that shit.
Done

Reply:
Cool
Always happy to help, de shit the code. HEHEH :)))